src/sequelize.ts

```diff
@@ -58,6 +58,26 @@ export function BigNumberStringType (propName: string): Partial<ModelAttributeCo
   }
 }
 
+/**
+ * consider that the field will be stored as string separated by '|' symbol in the data base,
+ * so you not be able to use array comparision when querying
+ * ArrayStringType for sequelize models
+ * @param propName
+ * @constructor
+ */
+export function ArrayStringType (propName: string): Partial<ModelAttributeColumnOptions> {
+  const separator = '|'
```

Comment:
why not comma `,`?

Reply:
I think about the symbol which is less used. For example, if you have an array of strings you probably can use comma in one of the elements